l10n_th_base_location/wizard/geonames_import.py

```diff
@@ -21,14 +21,44 @@ class CityZipGeonamesImport(models.TransientModel):
         [("th", "Thai"), ("en", "English")], string="Language", default="th"
```

Comment:
```suggestion
        [("th", "Thai"), ("en", "English")], string="Language of Thailand", default="th"
```
I think it specifications for Thailand. Could do you think about it?